modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -288,7 +288,11 @@ enum InterpolationFlags{
     - flag is __not__ set: \f$dst( \rho , \phi ) = src(x,y)\f$
     - flag is set: \f$dst(x,y) = src( \rho , \phi )\f$
     */
-    WARP_INVERSE_MAP     = 16
+    WARP_INVERSE_MAP     = 16,
+    /** flag, perform processing without trying to evaluate bit exact result using integral type calculations.
+    Inexact evaluation could be faster on specific platforms but could produce result the slightly differs b/w platforms.
+    */
+    EVAL_INEXACT         = 32
```

Comment:
I prefer to have `resizeFast()` instead of flag.
Also this would allow to minimize code binary size (like `cvtColor()` size hell).

Or add `resizeBitExact()/cv::bit_exact::resize()` instead because it doesn't support all current resize modes.